nekoyume/Assets/_Scripts/UI/Module/Inventory.cs

```diff
@@ -148,26 +143,32 @@ private void OnTabButtonClick(IToggleable toggleable, ItemType type, System.Acti
             itemSetDimPredicates?.ForEach(tuple =>
                 _dimConditionFuncsByItemType[tuple.type].Add(tuple.predicate));
 
-            if (isArena)
+            switch (inventoryType)
             {
-                var inventory = RxProps.PlayersArenaParticipant.Value.AvatarState.inventory;
-                SetInventory(inventory, onUpdateInventory);
+                case InventoryType.Default:
+                    ReactiveAvatarState.Inventory
```

Comment:
First of all, apologize for understanding of this code. could you tell me why subscribe only at this time?

Reply:
In other cases, we are looking at an inventory other than the existing inventory in the avatar State.

Reply:
> First of all, apologize for understanding of this code. could you tell me why subscribe only at this time?

https://github.com/planetarium/NineChronicles/pull/2051#discussion_r996544540